packages/lesswrong/lib/collections/comments/custom_fields.js

```diff
@@ -22,6 +22,7 @@ Comments.addField([
         resolver: async (comment, args, {currentUser, Users}) => {
           if (!comment.userId || comment.hideAuthor) return null;
           const user = await Users.loader.load(comment.userId);
+          if (user.deleted) return null;
```

Comment:
Do we want to add any indexes on this field?

Reply:
Indexes? Doesn't seem super relevant in this context, but might be missing the point. I associate the word `indexes` with MongoDB indexes, which seem irrelevant in this case.